x/concentrated-liquidity/swaps_test.go

```diff
@@ -593,29 +645,68 @@ func (s *KeeperTestSuite) TestCalcAndSwapOutAmtGivenIn() {
 
 				expectedLiquidity := math.GetLiquidityFromAmounts(DefaultCurrSqrtPrice, lowerSqrtPrice, upperSqrtPrice, test.poolLiqAmount0, test.poolLiqAmount1)
 				s.Require().Equal(expectedLiquidity.String(), updatedLiquidity.String())
+
+				if test.expectedLowerTickFeeGrowth != nil {
```

Comment:
I'm a bit worried about such checks because it is easy to forget with setting the value in the test case, and then we would skip validation. Can we require these values to always be set?

Reply:
sure, the only trade off here seeems like having more nil fields in the test case fields itself, but I do agree with you overall. Would remove these nil checks

Reply:
Added these for the first position ticks